pkg/acceptance/helpers/catalog_client.go

```diff
@@ -0,0 +1,51 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/stretchr/testify/require"
+)
+
+type CatalogIntegrationClient struct {
+	context *TestClientContext
+	ids     *IdsGenerator
+}
+
+func NewCatalogIntegrationClient(context *TestClientContext, idsGenerator *IdsGenerator) *CatalogIntegrationClient {
+	return &CatalogIntegrationClient{
+		context: context,
+		ids:     idsGenerator,
+	}
+}
+
+func (c *CatalogIntegrationClient) exec(sql string) error {
+	ctx := context.Background()
+	_, err := c.context.client.ExecForTests(ctx, sql)
+	return err
+}
+
+func (c *CatalogIntegrationClient) Create(t *testing.T) (sdk.AccountObjectIdentifier, func()) {
```

Comment:
please TODO to a new issue in SNOW-999142

Reply:
Done